### PR TITLE
Feat/prof page improvements

### PR DESCRIPTION
**Pull request by camrohwer**
Logout popup modal dialog for confirmation

### CODE REVIEW

File: apps/nextjs/src/app/profile/page.tsx [hunk 1]
```diff
@@ -27,7 +38,23 @@ export default async function Profile() {
...
         <div className="w-full max-w-2xl">
           <div className="flex h-1/6 max-h-40 flex-row justify-end rounded-b-2xl rounded-t-none bg-gradient-to-r from-primary to-muted p-4">
-            <LogoutButton />
+            <AlertDialog>
+              <AlertDialogTrigger>Log Out</AlertDialogTrigger>
+              <AlertDialogContent>
+                <AlertDialogHeader>
+                  <AlertDialogTitle>Log Out Confirmation</AlertDialogTitle>
+                  <AlertDialogDescription>
+                    Are you absolutely sure you want to Log Out?
```
Comment (anormananderson) [inline apps/nextjs/src/app/profile/page.tsx:47]:
```suggestion
                    Are you absolutely sure you want to log out?
```

**Review by anormananderson — COMMENTED** [review]
Are we worried at all about the discrepancy between functions named according "signing" in and out, whereas our dialogs refer to "logging" in and out?

(I can change this "Comment" into an "Approve," but I assumed you were looking for a review from someone who has previously touched the auth code.)

File: apps/nextjs/src/app/profile/page.tsx [hunk 2]
```diff
@@ -27,7 +38,23 @@ export default async function Profile() {
       <div className="flex min-h-screen flex-col items-center justify-center">
         <div className="w-full max-w-2xl">
           <div className="flex h-1/6 max-h-40 flex-row justify-end rounded-b-2xl rounded-t-none bg-gradient-to-r from-primary to-muted p-4">
-            <LogoutButton />
+            <AlertDialog>
+              <AlertDialogTrigger>Log Out</AlertDialogTrigger>
```
Comment (callumcurtis) [inline apps/nextjs/src/app/profile/page.tsx:42]: I'd like to see a logout icon ([example](https://www.google.com/search?sca_esv=6459c9e3b5d5a0f2&q=logout+icon&tbm=isch&source=lnms&sa=X&ved=2ahUKEwjCm8aTt9GEAxXCHTQIHTrKDCoQ0pQJegQIEBAB&biw=1440&bih=2399&dpr=1#imgrc=tlNev458ccsWWM), corresponds to "Exit" in Radix Icons) tried instead

File: packages/ui/src/button.tsx [hunk 3]
```diff
@@ -20,6 +20,7 @@ const buttonVariants = cva(
           "hover:bg-secondary/80 bg-secondary text-secondary-foreground shadow-sm",
         ghost: "hover:bg-accent hover:text-accent-foreground",
         link: "text-primary underline-offset-4 hover:underline",
+        transparent: "bg-transparent text-primary-foreground",
```
Comment (callumcurtis) [inline packages/ui/src/button.tsx:23]: If you go with the icon approach you could remove this as it wouldn't be needed

**Review by callumcurtis — CHANGES_REQUESTED** [review]
Looks good. Small suggestion about using an icon.

**Pull request closed without merging**